governance.md

```diff
@@ -0,0 +1,363 @@
+# Main Governance Document
+
+The official version of this document, along with a list of
+individuals and institutions in the roles defined in the governance
+section below, is contained in The Project Governance Repository at:
+
+[https://github.com/dask/governance](https://github.com/dask/governance)
+
+The Project
+===========
+
+The Dask Project (The Project) is an open source software project.
+The goal of The Project is to
+develop open source software and enable parallel computing within the PyData
+software ecosystem.  The Software developed
+by The Project is released under the BSD-3-Clause (or similar) open source license,
+developed openly and hosted in public GitHub repositories under the
+[Dask GitHub organization](https://github.com/dask). Examples of
+Project Software include the Dask core library, Dask Array, Dask DataFrame, the
+Dask distributed scheduler, libraries to deploy Dask on various resource
+managers, and some extensions to other domains like machine learning.
+The Services run by the
+Project consist of public websites and web-services that are hosted
+under the dask.org domain.
+
+The Project is developed by a team of distributed developers, called
```

Comment:
Nitpick: change to “distributed team of developers” since we have a project named distributed :)